save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/codeeditor/SandboxCodeEditorComponent.kt

```diff
@@ -169,7 +169,6 @@ private fun sandboxCodeEditorComponent() = FC<SandboxCodeEditorComponentProps> {
                     if (window.confirm("Some changes are not saved. Save and run execution?")) {
                         uploadTexts()
                         props.doRunExecution()
-                        window.alert("Successfully saved and started execution.")
                     } else {
                         window.alert("Run canceled.")
                     }
```

Comment:
This movement will cause double alert if there were no changes. Please fix it

Reply:
yes, already found -- fixing it now